### DISCUSSION TITLE
Long dotted paths in Previous/Next links protude

### DISCUSSION THREAD

**Discussion by flying-sheep** [pradyunsg/furo#381] [Potential Issue] [answered]
Looks like this:

![grafik](https://user-images.githubusercontent.com/291575/155317783-7587084e-b5ae-4a1d-b84f-d972c93c27b0.png)

https://docs.scvi-tools.org/en/latest/api/reference/scvi.dataloaders.SemiSupervisedDataSplitter.html

I think it would be best to insert zero width spaces at dots or so, like in https://github.com/pradyunsg/furo/pull/382

**Comment by pradyunsg** [comment, marked as answer]
Ah, nice catch!

45fa3ca73cca8d95d9b6991e4ca588baa9e4a0c3 fixes this by allowing the "word" to be broken to wrap text. :)
